hasql/asyncsqlalchemy.py

```diff
@@ -108,15 +108,16 @@ def async_sessionmaker(
     @asynccontextmanager
     async def create_async_session() -> AsyncIterator[AsyncSession]:
         """Create an async session with connection from the pool."""
-        # TODO(PY310): Use parentheses to break the statement in multiple lines
-        async with pool_manager.acquire(**acquire_kwargs) as connection, \
-        class_(
-            bind=connection,
-            autoflush=autoflush,
-            expire_on_commit=expire_on_commit,
-            info=info,
-            **kw,
-        ) as session:
+        async with (
+            pool_manager.acquire(**acquire_kwargs) as connection,
+            class_(
```

Comment:
what do you think about rename it as `klass`?

Reply:
https://github.com/aiokitchen/hasql/pull/21#discussion_r2242883923